index.js

```diff
@@ -46,21 +46,28 @@ function isFile(file) {
 
 var getStat = function (opts) {
     var stats = opts.stats || process.env.BROWSERSLIST_STATS;
-    if ( stats ) return stats;
 
-    if ( typeof opts.path === 'undefined' ) opts.path = '.';
-    var dirs = path.resolve(opts.path).split(path.sep);
+    if ( !stats ) {
```

Comment:
Marge is broken (it is always good to check it after sending a PR). Don’t worry, I will pick up right changes manually.